aws-ts-apigateway-auth0/index.ts

```diff
@@ -0,0 +1,131 @@
+// Copyright 2016-2018, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import * as awsx from "@pulumi/awsx";
+import * as pulumi from '@pulumi/pulumi';
+
+import * as jwksClient from 'jwks-rsa';
+import * as jwt from 'jsonwebtoken';
+import * as util from 'util';
+
+const config = new pulumi.Config();
+const jwksUri = config.require("jwksUri");
+const audience = config.require("audience");
+const issuer = config.require("issuer");
+
+// Create the Lambda Authorizer
+const authorizers: awsx.apigateway.LambdaAuthorizer[] = [
+    awsx.apigateway.getTokenLambdaAuthorizer({
+        authorizerName: "jwt-rsa-custom-authorizer",
+        header: "Authorization",
+        handler: async (event: awsx.apigateway.AuthorizerEvent): Promise<awsx.apigateway.AuthorizerResponse> => {
```

Comment:
your `handler` property is already strongly typed, right?  so we can avoid the types here.  Unless you think it's really important to have them for clarity.